app/views/transactions/index.njk

```diff
@@ -28,7 +28,11 @@ Transactions - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV
   {% include "transactions/display-size.njk" %}
 
   <h3 class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-top-3" id="total-results">
-    {{totalFormatted}} transactions
+    {% if totalOverLimit %}
+      Over {{csvMaxLimitFormatted}} transactions
+    {% else %}
+      {{totalFormatted}} transactions
+    {% endif %}
```

Comment:
As the `transaction-view` utility has been updated I think this change will also need to be made in the `all-services-transactions/index.njk` as well. 

Unfortunately they use slightly different base templates because of some quirk at the time if I remember correctly; it would be good for us to try and remove that duplication in a future PR if possible.

Reply:
Good point! updated.